tests/discovery/processing/test_experiment_preprocessor.py

```diff
@@ -0,0 +1,211 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from slamd.common.error_handling import ValueNotSupportedException, SequentialLearningException, \
+    SlamdUnprocessableEntityException
+from slamd.discovery.processing.algorithms.experiment_preprocessor import ExperimentPreprocessor
+from slamd.discovery.processing.models.experiment_data import ExperimentData
+from slamd.discovery.processing.models.model_type import ModelType
+
+
+def create_valid_experimentdata():
+    """
+    Creates a valid ExperimentData object for ExperimentPreprocessor.validate_experiment() tests.
+    Do not use for other tests - the Dataframes are filled with dummy data.
+    """
+    feature_names = ['feature1', 'feature2', 'feature3']
+    target_names = ['target1', 'target2', 'target3']
+    apriori_names = ['apriori1', 'apriori2', 'apriori3']
+
+    df = pd.DataFrame(columns=feature_names + target_names + apriori_names)
+
+    for col in feature_names+apriori_names:
+        df[col] = np.arange(10)
+
+    df.loc[1, target_names] = [1, 2, 3]
+    df.loc[4, target_names] = [4, 5, 6]
+    df.loc[5, target_names] = [6, 7, 8]
+
+    return ExperimentData(
+        dataframe=df,
+        model=str(ModelType.GAUSSIAN_PROCESS.value),
+        curiosity=2,
+
+        target_names=target_names,
+        target_weights=[1, 2, 1],
+        target_thresholds=[None, 5, 6],
+        target_max_or_min=['max', 'max', 'min'],
+
+        apriori_names=apriori_names,
+        apriori_weights=[1, 2, 1],
+        apriori_thresholds=[None, 5, 6],
+        apriori_max_or_min=['max', 'max', 'min'],
+
+        feature_names=feature_names
+    )
+
+
+def test_validate_experiment_valid():
```

Comment:
I think we can add a small comment explaining that no assertion is needed here as we simply want to check that the code runs without raising any exceptiona at all

Reply:
Added a comment